estate_account/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import api, fields, models
```

Comment:
```suggestion
from odoo import Command, api, fields, models
```
And below you use Command.create({}) instead. So that your code is less bloated.Command.creat :slightly_smiling_face: